collect_app/src/main/java/org/odk/collect/android/formhierarchy/QuestionAnswerProcessor.kt

```diff
@@ -25,7 +25,12 @@ object QuestionAnswerProcessor {
             return ""
         }
 
-        if (!fep.answerText.isNullOrBlank() &&
+        val answerText = try {
```

Comment:
Due to the nature of how external choices are handled by the `search` function, there are some cases we can't fully resolve. One of them is described here: https://github.com/getodk/collect/issues/6801#issuecomment-3056718884

For example, if we open a blank form where a non-first select question uses the last-saved instance and immediately access the hierarchy view, the hierarchy attempts to determine the answer to display before the choices have been loaded (which only happens when the question is actually displayed). In older versions of the app, this led to an error.

With this change, we now display the underlying XML value until the choices are fully loaded.